libs/common/src/response-messages/index.ts

```diff
@@ -181,7 +181,8 @@ export const ResponseMessages = {
     },
     ledger: {
         success: {
-            fetch: 'Ledgers retrieved successfully.'
+            fetch: 'Ledgers retrieved successfully.',
```

Comment:
Please change the messages, we are not fecthing the ledger directly

Reply:
Resolved